CHANGELOG.md

```diff
@@ -53,8 +53,7 @@
 - Fixed a bug in local testing mode that caused a column to contain None when it should contain 0
 - Fixed a bug in `StructField.from_json` that prevented TimestampTypes with `tzinfo` from being parsed correctly.
 - Fixed a bug in function `date_format` that caused an error when the input column was date type or timestamp type.
-- Fixed a bug in dataframe that null value can be inserted in a non-nullable column.
```

Comment:
I think you don't mean to remove this changelog right?

Reply:
oops. will fix that.